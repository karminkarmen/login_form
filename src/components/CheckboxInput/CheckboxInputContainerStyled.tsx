import styled from 'styled-components';
import { centerized } from '../../configs/mixins';

const CheckboxInputContainerStyled = styled.div`
    ${centerized}
    margin: ${props => props.theme.margins.normal};
`;

export default CheckboxInputContainerStyled;