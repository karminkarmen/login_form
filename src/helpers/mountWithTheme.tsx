import React from 'react';
import { mount } from 'enzyme';
import { ThemeProvider } from 'styled-components';
import theme from '../configs/theme';

const mountWithTheme = (child: React.ReactElement) => {
    return mount(child, {
        wrappingComponent: ({ children }) => <ThemeProvider theme={theme}>{children}</ThemeProvider>,
    });
}

export default mountWithTheme;