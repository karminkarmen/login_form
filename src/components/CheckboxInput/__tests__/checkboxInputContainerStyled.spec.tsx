import React from 'react';
import CheckboxInputContainerStyled from '../CheckboxInputContainerStyled';
import mountWithTheme from '../../../helpers/mountWithTheme';

describe('<CheckboxInputContainerStyled />', () => {

    it('matches snapshot', () => {
        const wrapper = mountWithTheme(<CheckboxInputContainerStyled />);
        expect(wrapper.html()).toMatchSnapshot();
    });

});
