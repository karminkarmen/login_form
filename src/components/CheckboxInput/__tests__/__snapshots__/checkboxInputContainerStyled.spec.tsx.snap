// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckboxInputContainerStyled /> matches snapshot 1`] = `
"<styled.div>
  <div className=\\"sc-AxjAm drhBhZ\\" />
</styled.div>"
`;
